src/AccountProfile.jsx

```diff
@@ -1,32 +1,11 @@
-const accountId = props.accountId || context.accountId;
-const blockHeight = props.blockHeight;
-const blockTimestamp = props.blockTimestamp;
-const profileUrl = `/${REPL_ACCOUNT}/widget/ProfilePage?accountId=${accountId}`;
-const verifications = props.verifications;
-const showFlagAccountFeature = props.showFlagAccountFeature ?? false;
-const profile = props.profile || Social.get(`${accountId}/profile/**`, "final");
-
-function returnProfileForUser(user) {
-  const rawImage = user.profile_image_1 || user.profile_image_2 || user.profile_image_3;
-  const image = rawImage && rawImage.indexOf("http") === 0 ? { url: rawImage } : { ipfs_cid: rawImage };
-  const name = user.profile_name ?? "";
-  let tags = null;
-
-  if (user.profile_tags) {
-    tags = {};
-    user.profile_tags.forEach((tag) => (tags[tag] = ""));
-  }
+let { accountId, blockHeight, blockTimestamp, profile, verifications, showFlagAccountFeature } = props;
 
-  if (image && tags) {
-    return {
-      image,
-      name,
-      tags,
-    };
-  }
+accountId = accountId || context.accountId;
+showFlagAccountFeature = showFlagAccountFeature ?? false;
+profile = profile || Social.get(`${accountId}/profile/**`, "final");
+
+const profileUrl = `/${REPL_ACCOUNT}/widget/ProfilePage?accountId=${accountId}`;
 
-  return null;
-}
```

Comment:
I dropped this method because from now QueryAPI returns `image` and `tags` in format we need. Also I noticed that `Social.get(`${accountId}/profile/**`, "final")` doesn't have a `profile_image_1` or `profile_image_2` and returns `image` prop (as an example) in format:
```
image: {
  ipfs_cid: "asdasdasdasdasdasdasd"
}
or
image: {
  nft: {
    contractId: "asasdasda",
    tokenId: "asdasdasdasd"
  }
}
```
and I don't see any reason to use this method here